docs/sql-ref-syntax.md

```diff
@@ -19,4 +19,64 @@ license: |
   limitations under the License.
 ---
 
-Spark SQL is Apache Spark's module for working with structured data. The SQL Syntax section describes the SQL syntax in detail along with usage examples when applicable.
+Spark SQL is Apache Spark's module for working with structured data. The SQL Syntax section describes the SQL syntax in detail along with usage examples when applicable. This document provides a list of Data Definition and Data Manipulation Statements, as well as Data Retrieval and Auxiliary Statements.
+
+### Data Definition Statements
+- [ALTER DATABASE](sql-ref-syntax-ddl-alter-database.html)
+- [ALTER TABLE](sql-ref-syntax-ddl-alter-table.html)
+- [ALTER VIEW](sql-ref-syntax-ddl-alter-view.html)
+- [CREATE DATABASE](sql-ref-syntax-ddl-create-database.html)
+- [CREATE FUNCTION](sql-ref-syntax-ddl-create-function.html)
+- [CREATE TABLE](sql-ref-syntax-ddl-create-table.html)
+- [CREATE VIEW](sql-ref-syntax-ddl-create-view.html)
+- [DROP DATABASE](sql-ref-syntax-ddl-drop-database.html)
+- [DROP FUNCTION](sql-ref-syntax-ddl-drop-function.html)
+- [DROP TABLE](sql-ref-syntax-ddl-drop-table.html)
+- [DROP VIEW](sql-ref-syntax-ddl-drop-view.html)
+- [REPAIR TABLE](sql-ref-syntax-ddl-repair-table.html)
+- [TRUNCATE TABLE](sql-ref-syntax-ddl-truncate-table.html)
+- [USE DATABASE](sql-ref-syntax-qry-select-usedb.html)
+
+### Data Manipulation Statements
+- [INSERT INTO](sql-ref-syntax-dml-insert-into.html)
+- [INSERT OVERWRITE](sql-ref-syntax-dml-insert-overwrite-table.html)
+- [INSERT OVERWRITE DIRECTORY](sql-ref-syntax-dml-insert-overwrite-directory.html)
+- [INSERT OVERWRITE DIRECTORY with Hive format](sql-ref-syntax-dml-insert-overwrite-directory-hive.html)
+- [LOAD](sql-ref-syntax-dml-load.html)
+
+### Data Retrieval Statements
+- [CLUSTER BY Clause](sql-ref-syntax-qry-select-clusterby.html)
+- [DISTRIBUTE BY Clause](sql-ref-syntax-qry-select-distribute-by.html)
+- [EXPLAIN](sql-ref-syntax-qry-explain.html)
+- [GROUP BY Clause](sql-ref-syntax-qry-select-groupby.html)
+- [HAVING Clause](sql-ref-syntax-qry-select-having.html)
+- [LIMIT Clause](sql-ref-syntax-qry-select-limit.html)
+- [ORDER BY Clause](sql-ref-syntax-qry-select-orderby.html)
+- [SORT BY Clause](sql-ref-syntax-qry-select-sortby.html)
+- [WHERE Clause](sql-ref-syntax-qry-select-where.html)
```

Comment:
We need these links to the sub-categoris of SELECT? Just linking to `sql-ref-syntax-qry-select.md` here is not enough?

Reply:
Thanks for your comment. I want to list all the sql statements on one page, so I prefer to have all the links here.